RecoBTag/Combined/python/combinedMVAV2_cff.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Please put the content of this cff file into `combinedMVA_cff.py`. We have too many configuration files already ;)